homeassistant/components/zwave/config_flow.py

```diff
@@ -12,6 +13,22 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+VENDOR_IDS = (('Aeon Labs Gen5', 'VID_0658'),)
+
+
+def _get_z_stick():
+    import serial.tools.list_ports
+
+    for port in serial.tools.list_ports.comports(include_links=False):
+        if port.vid is None:
+            continue
+        for vendor, vid in VENDOR_IDS:
```

Comment:
So if you were looking to support other USB sticks, there might be some special exceptions to this simple logic, for example the HUSBZB-1 Z-Wave/Zigbee combo USB stick. It has a path for both the Z-Wave and Zigbee devices, and they both share the same VID, so which one would be chosen? You might return the Zigbee device path. Not a problem now, but more of an FYI.

On the plus side, I can confirm this does work well on Mac OS. Pyserial only returns a `/dev/cu.usbmodemFA131` path for my UZB3 stick. The alternative path`/dev/tty.usbmodemFA131` device, won't work and isn't listed.